deepmatcher/data/process.py

```diff
@@ -251,34 +270,35 @@ def process_unlabeled(path, trained_model, ignore_columns=None):
     column_naming['label'] = None
 
     fields = _make_fields(header, column_naming['id'], column_naming['label'],
-                          ignore_columns, train_info.lowercase, train_info.tokenize,
-                          train_info.include_lengths)
+                          ignore_columns, train_info.lowercase,
+                          train_info.tokenize, train_info.include_lengths)
 
     begin = timer()
     dataset_args = {'fields': fields, 'column_naming': column_naming}
-    dataset = MatchingDataset(path=path, **dataset_args)
+    dataset = MatchingDataset(stream=stream, **dataset_args)
```

Comment:
Maybe I'm missing something - MatchingDataset does not take "stream" as argument right?